Update quantized aot_lib's visibility targets to include ExecuTorch clients

luyich opened the pull request:
As titled.

Differential Revision:
D69572954

Privacy Context Container: L1281892

mcr229 commented on kernels/quantized/targets.bzl:
```diff
@@ -47,7 +47,7 @@ def define_common_targets():
         name = "aot_lib",
         yaml_target = ":quantized.yaml",
-        visibility = ["//executorch/..."],
+        visibility = ["//executorch/...","@EXECUTORCH_CLIENTS"],
```
```suggestion
        visibility = [
            "//executorch/...",
            "@EXECUTORCH_CLIENTS",
       ],
```

mcr229 reviewed (approved):
stamping to unblock, but please take a look at the nit

luyich committed:
minymal llama backbone export (#8453)

Summary:



Export llama 1b from minymal (for testing purpose fixed shape of 1*len*2048) is used. This is build on based on the llava example

Reviewed By: asleep, mcr229

Differential Revision: D69572954
